setup.py

```diff
@@ -25,7 +25,7 @@
     package_data={"spidermon": ["VERSION"]},
     zip_safe=False,
     include_package_data=True,
-    install_requires=["jsonschema[format]", "python-slugify", "six>=1.10.2"],
+    install_requires=["jsonschema[format]", "python-slugify", "six>=1.11.0"],
```

Comment:
Bad luck with `bumpversion`?

Reply:
Yes. But the current version is 1.12.0, so it is not a big problem.